pkgs/development/tools/git-ftp/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, pandoc, man-db }:
+stdenv.mkDerivation rec {
+  name = "git-ftp-${version}";
+  version = "1.4.0";
+  src = fetchFromGitHub {
+    owner = "git-ftp";
+    repo = "git-ftp";
+    rev = "2c22f5118d61070449532a524db77bf810c13b97";
+    sha256 = "0n8q1azamf10qql8f8c4ppbd3iisy460gwxx09v5d9hji5md27s3";
+  };
+
+  # fixupPhase is needed for manpages
+  phases = ["unpackPhase" "installPhase" "fixupPhase"];
+
+  installPhase = ''
+    make install${lib.optionalString (!stdenv.isDarwin) "-all"} prefix=$out
+  '';
+
+  buildInputs = lib.optionals (!stdenv.isDarwin) [pandoc man-db];
```

Comment:
man-db builds only on `platforms.linux` and i thought since there is no man-db there is no point in building the man pages on Darwin.